src/Files.App/Utils/Archives/ArchiveCreator.cs

```diff
@@ -123,6 +124,9 @@ public ArchiveCreator()
 		/// <inheritdoc/>
 		public string GetArchivePath(string suffix = "")
 		{
+			if (string.IsNullOrEmpty(Directory))
+				throw new NullReferenceException($"{Directory} is null.");
```

Comment:
I don't think we should throw in release environments, returning early is better.